### DISCUSSION TITLE
Theme

### DISCUSSION THREAD

**Discussion by fadri14** [dooit-org/dooit#125] [General]
Hello everyone,

I love the project however I would like to change the style a bit. Can you please tell me how to make the app background transparent so i can see my pc wallpaper? Thanks to anyone who takes the time to respond.

**Comment by kraanzu** [comment]
Heyy @fadri14, I'm not sure but after textual version 0.10, the transparency was gone for some terminals :(
For me, there was no transparent screen in alacritty but kitty supported transparency. Not sure what's the underlying issue :(

Can you try that?

**Comment by fadri14** [comment]
I'm sure alacrity. So that means that there is a configuration for that but it's not sure to work, is that it? How should I write it in the configuration file?

**Comment by kraanzu** [comment]
Here's the config for [kitty](https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.background_opacity) and [alacritty](https://github.com/alacritty/alacritty/issues/5470#issuecomment-1021957039)

**Reply by fadri14** [reply]
thank you
